### ISSUE TITLE
Future warning about `decode_timedelta`

### ISSUE THREAD

**Issue by xylar** [MPAS-Dev/MPAS-Analysis#1062]
```
/gpfs/fs1/home/ac.xylar/mpas_work/MPAS-Analysis/fix-bsf/mpas_analysis/shared/climatology/mpas_climatology_task.py:290: FutureWarning: In a future version of xarray decode_timedelta will default to False rather than None. To silence this warning, set decode_timedelta to True, False, or a 'CFTimedeltaCoder' instance.
  with xarray.open_dataset(self.inputFiles[0]) as ds:
```

We need to deal with this before it becomes a problem.

**Comment by xylar** [comment]
These warnings are annoying but I think we can ignore them.  We don't want to decode times as TimeDeltas so we're fine with the new behavior.
